serverless.yml

```diff
@@ -3,21 +3,59 @@ service: media-alpha
 provider:
   name: aws
   runtime: python3.6
-  profile: MyZebraUser-staging
-  region: us-west-1
+  region: us-east-1
+  stage: ${opt:stage, 'dev'}
+  environment:
+    API_TOKEN: ${self:custom.stages.${self:provider.stage}.API_TOKEN}
+    PLACEMENT_ID: ${self:custom.stages.${self:provider.stage}.PLACEMENT_ID}
+    CI_REPO_NAME: ${git:repository}
+    CI_COMMIT_ID: ${git:commit}
+    CI_BRANCH: ${git:branch}
 
 functions:
   phone_number:
     handler: handler.get_phone_number
     events:
       - http:
-          path: ''
+          path: 'phone-number'
           method: POST
+          cors: true
+  versioning:
+    handler: healthcheck.check_version
+    events:
+      - http:
+          path: 'version'
+          method: GET
+          cors: true
 
 plugins:
-  - serverless-offline-python
+  - serverless-offline
   - serverless-python-requirements
+  - serverless-domain-manager
+  - serverless-plugin-git-variables
+
+package:
+  individually: true
+  exclude:
+    - .virtualenv/**
+    - .venv/**
```

Comment:
I guess we should stick with `.virtualenv` to stay consistent. Let's remove `.env` and `venv`